src/components/app/data/hooks/useRecommendCoursesForMe.js

```diff
@@ -1,49 +1,55 @@
-import {
-  useCallback,
-  useContext,
-  useMemo,
-  useState,
-} from 'react';
 import { useParams, useMatch } from 'react-router-dom';
-import { AppContext } from '@edx/frontend-platform/react';
-import { logInfo } from '@edx/frontend-platform/logging';
 
-import { useEnterpriseCuration } from '../../../search/content-highlights/data';
+// import { useEnterpriseCuration } from '../../../search/content-highlights/data';
```

Comment:
[inform] TODO: need to figure out the commented out parts still... `isAssignmentOnlyLearner` logic needs to also be computed within the `SiteHeader`, so TBD where this logic will live; first hunch is a new custom hook `useIsAssignmentsOnlyLearner()` that returns true/false based on calling the `useEnterpriseCustomerUserSubsidies()` hook to get at the requisite subsidies-related data to make that determination.